docs/user_guide.md

```diff
@@ -116,72 +116,72 @@ CodeChecker check -w ~/codechecker_wp -n myProject -l ~/codechecker_wp/build_log
                         related data. (default: /home/<user_home>/.codechecker)
   -n NAME, --name NAME  Name of the analysis.
   -b COMMAND, --build COMMAND
-                        Build command which is used to build the project
+                        Build command which is used to build the project.
   -l LOGFILE, --log LOGFILE
-                        Path to the log file which is created during the
-                        build. If there is an already generated log file with
-                        the compilation commands generated by 'CodeChecker
-                        log' or 'cmake -DCMAKE_EXPORT_COMPILE_COMMANDS'
-                        CodeChecker check can use it for the analysis in that
-                        case running the original build will be left out from
-                        the analysis process (no log is needed).
-  -j JOBS, --jobs JOBS  Number of jobs. Start multiple processes for faster
-                        analysis (default: 1)
-                        -u SUPPRESS, --suppress SUPPRESS
-                        Path to suppress file. Suppress file can be used to
-                        suppress analysis results during the analysis. It is
-                        based on the bug identifier generated by the compiler
-                        which is experimental. Do not depend too much on this
-                        file because identifier or file format can be changed.
-                        For other in source suppress features see the user
-                        guide.
+                        Path to the log file which is created
+  -j JOBS, --jobs JOBS  Number of jobs.Start multiple processes for faster
+                        analysis. (default: 1)
+  -u SUPPRESS, --suppress SUPPRESS
+                        Path to suppress file. Suppress file can be used
   -c [DEPRECATED], --clean [DEPRECATED]
                         DEPRECATED argument! (default: None)
   --update [DEPRECATED]
                         DEPRECATED argument! (default: None)
   --force               Delete analysis results form the database if a run
-                        with the given name already exists (default: False)
+                        with the given name already exists. (default: False)
   -s SKIPFILE, --skip SKIPFILE
                         Path to skip file.
+  --quiet-build         Do not print out the output of the original build.
+                        (default: False)
+  --add-compiler-defaults
+                        Fetch built in compiler includepaths and defines and
+                        pass them to Clang. This isuseful when you do cross-
+                        compilation. (default: False)
   -e ENABLE, --enable ENABLE
                         Enable checker.
   -d DISABLE, --disable DISABLE
                         Disable checker.
-  --keep-tmp            Keep temporary report files generated during the
+  --keep-tmp            Keep temporary report filesgenerated during the
```

Comment:
There are some bogus prints here, a newline in the source code was not followed by a ` `.